### PR TITLE
msm camera: force power up/down settings for imx234/imx268 from h850 …

### PR DESCRIPTION
**Pull request by ChristianFrisson**
Power up/down values obtained from v10E stock logs.

Please inform me how to improve the pull request if it can not be merged as is.

### CODE REVIEW

**Comment by Rashed97** [comment]
Oh damn. Nice. Do they power on now?

**Comment by ChristianFrisson** [comment]
This is a screenshot taken when using Snap on CM 13.0 with the rear camera aiming at a mirror so that the phone backside is visible:
screenshot_20161015-191705

File: drivers/media/platform/msm/camera_v2/sensor/msm_sensor_driver.c [hunk 1]
```diff
@@ -35,6 +35,62 @@ static int32_t msm_sensor_driver_platform_probe(struct platform_device *pdev);
 /* Static declaration */
 static struct msm_sensor_ctrl_t *g_sctrl[MAX_CAMERAS];
 
+// Power up/down settings obtained from LGH850v10E logs
+// Should be moved into a driver file like imx234.c
+// imx234.c settings do not seem to be parsed 
+// even when "obj-$(CONFIG_IMX234) += imx234.o" is enabled in the Makefile
```
Comment (Rashed97) [inline drivers/media/platform/msm/camera_v2/sensor/msm_sensor_driver.c:41]: The sensor specific drivers (ex. imx234) are no longer used. msm_sensor probe's all the cameras now.

**Comment by Rashed97** [comment]
Nice, I'll run a CM13 build later today with these (I'm on CM14 atm), about to have guests over. Do you have a Hangouts that we can chat over?

**Pull request merged**